tools/clang/test/HLSLFileCheck/hlsl/diagnostics/warnings/unused-output-target-warning.hlsl

```diff
@@ -0,0 +1,18 @@
+// RUN: %dxc -T ps_6_0 %s | FileCheck -input=stderr %s
+
+// CHECK-NOT: warning: Declared output SV_Target0 not fully written in shader.
+// CHECK: warning: Declared output SV_Target1 not fully written in shader.
+// CHECK: warning: Declared output SV_Target2 not fully written in shader.
+// CHECK: warning: Declared output SV_Target3 not fully written in shader.
+
+void main(out float4 outRT0
+    : SV_Target0, out float4 outRT1
+    : SV_Target1, out float4 outRT2
+    : SV_Target2, out float4 outRT3
+    : SV_Target3) {
```

Comment:
I suspect this fell victim to some autoformatting that makes it confusing to read. I think it's worth changing it so that the SV_Target*s are on the same line as the parameters they apply to.